### DISCUSSION TITLE
making rethink dns desktop ver/ chrome extension ?

### DISCUSSION THREAD

**Discussion by heartlog** [celzero/rethink-app#945] [Ideas]
as far as I have used rethinkDNS, its good !! 
I wish there was an extension of rethinkdns for desktop (like adguard).
please consider about it.
Thanks..

**Comment by ignoramous** [comment]
We are a team of 2, out of money, and barely making any revenue with Rethink; and just one developer on Android right now.

We may yet go cross-platform (with Linux more likely), but that isn't something we are even thinking about right this moment. Someday. Also, AdGuard is a plenty capable alternative, imo (:

**Reply by NotMainstream** [reply]
Why don`t you try to apply for a grant from Fedora to hire some help ? There are no good workstation firewalls with functions as awesome as Rethink for Linux that are free. There was Portmaster that barely made the job until they decided to not only ask money for their VPN service (which was fine) but also for crucial parts of the app that were free before like investigating the connections history which transformed then in another greedy company that flipped the table on their userbase.  Congratulations for proving that two devs can create an app that is better than any other big company can come up with !

**Reply by ignoramous** [reply]
Thanks for your kind words.

> There are no good workstation firewalls with functions as awesome as Rethink for Linux that are free.

Check out [opensnitch](https://github.com/evilsocket/opensnitch), if you haven't already. I don't use it anymore, but it was one such inspiration for Rethink.

**Comment by heartlog** [comment]
It's alright! I primarily use my PC. If there is any consideration for implementing a subscription-based model, you could take inspiration from services like AdGuard and NextDNS, but with additional customization options and control features.
I would be delighted to use such a model.

**Reply by ignoramous** [reply]
That's the plan. We are starting with RPN, a super cheap, FOSS, proxy network: #946
